### DISCUSSION TITLE
WHO THE HECK POSTED DIS 🤮 🤮 🤮 🤮 🤮 🤮 🤮 🤮 🤮 🤮 🤮

### DISCUSSION THREAD

**Discussion by Outfit8TSB** [GAME-CLI-SRV-DEV/GCSD-Discussions#16] [General]
reddit post from r/griefcraft
# approximasteranarchy2004.falixsrv.me:20517
14 year old who things he is a king

Destroy the poster's r/place drawing

**Comment by juul-vereijken** [comment]
let me hit the gym first

**Reply by Outfit8TSB** [reply]
demonview you obese
you are not allowed to post this

**Comment by Outfit8TSB** [comment]
Also what the heck psoted this
u/anarchyac
# Greif This
approximasteranarchy2004.falixsrv.me:20517

**Reply by Outfit8TSB** [reply]
You know what i am gonna do if you post this??!?!?!!??!!😈😈😈😈😈

i am pressing the Red Nuke Button

🔴 ←←← PRESS 🔴 TO NUKE ANARCHY.AC

**Reply by Outfit8TSB** [reply]
I also got nuclear password so
```
telnet nuke.approximaster.gov_42069_Bedrock-1.69.420.viaproxy.anarchyconnect.ddnsgeek.com:56114
Pass>d0wn1n0h10

Nuclear Launch Ready.
What Launcher Will You Use?
Launcher-1-1=NearLockerOfLosers
Launcher-1-2=NearWeakTremblingFortress
Launcher1-3=NearViaProxyAnarchyConnectDuckdnsorg56114HQ
Launcher1=LAUNCH EVERYWHERE!!!!
>Launcher1
Press 🔴 TO NUKE!!!!
```
🔴🔴🔴🔴🔴🔴🔴🔴🔴


```
